config/portals.yml

```diff
@@ -64,6 +64,19 @@ portals: &portals
     scripts:
       - edsc-portal.cwic.js
 
+  IDN:
+    org: CEOS IDN
+    title: International Directory Network (IDN) Search
+    params:
+    	keyword:
+    		- *
```

Comment:
This is invalid yml, needs to be less spaces here.

Reply:
@mmorahan this still needs to be cleaned up. If you have made the change locally, please push that commit. I'll move this back to "unresolved" for the moment since the change isn't on Github yet.

Reply:
@mmorahan this is still invalid, there are too many spaces on line 72